kotsadm/operator/pkg/applier/process.go

```diff
@@ -25,8 +26,23 @@ func Run(cmd *exec.Cmd) ([]byte, []byte, error) {
 		return nil, nil, errors.Wrap(err, "failed to start commmand")
 	}
 
-	stdout, _ := ioutil.ReadAll(stdoutReader)
-	stderr, _ := ioutil.ReadAll(stderrReader)
+	var stdout, stderr []byte
+	var wg sync.WaitGroup
+
+	wg.Add(1)
```

Comment:
Unrelated.  But I think `stdoutReader` got blocked at some point because `stderrReader` buffer got filled up and the command never terminated.